cloud/getting-started/cc-trial.md

```diff
@@ -0,0 +1,38 @@
+---
+id: centreon-cloud-trial
+title: About the Centreon Cloud trial
+---
+
+## What is the Centreon Cloud trial?
+
+Centreon Cloud trial is a **free 7-day license** that includes the features available with our [IT Edition](https://www.centreon.com/centreon-editions/#editions-table) and with a monitoring capacity of up to 125 hosts.
+
+Centreon Cloud offers the Centreon monitoring platform in a SaaS version, for an experience free of installation and maintenance.
+
+> Visit the [Editions](https://www.centreon.com/centreon-editions/) page to find out about the features included in the various editions of Centreon.
+
+## About users and organizations
+
+### User accounts
+
+User accounts are managed in the [Centreon CIAM](../ciam/ciam.md) (Customer Identity and Access Management) module, Centreon’s access management tool. The person who first requests the Cloud Trial using the form becomes the "administrator". They will invite other users to the platform.
```

Comment:
Désolé mais je ne suis pas fan de la formulation.
Perso je dirais plus qqch comme
The person requesting for a trial becomes de facto Centreon platform administrator, and as such is entitled to invite other user to join.
Après je vous laisse juge